src/Core/src/Platform/Android/TransformationExtensions.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using AView = Android.Views.View;
+using ViewHelper = Com.Microsoft.Maui.ViewHelper;
 
 namespace Microsoft.Maui
 {
 	public static class TransformationExtensions
 	{
 		public static void UpdateTranslationX(this AView nativeView, IView view)
 		{
+			if (!view.Initialized)
+				return;
```

Comment:
This is still WIP, I want to see if I can come up with a better idea than putting this in a bunch of places...